sentry_sdk/integrations/django/__init__.py

```diff
@@ -319,6 +320,34 @@ def _patch_django_asgi_handler():
     patch_django_asgi_handler_impl(ASGIHandler)
 
 
+def _set_transaction_name_and_source(scope, transaction_style, request):
```

Comment:
same comment above about not returning scope since it's mutated anyway

Reply:
done